test-configs.yaml

```diff
@@ -461,6 +461,24 @@ device_types:
     filters:
       - blacklist: {defconfig: ['allnoconfig', 'allmodconfig']}
 
+  meson_g12a_sei510:
+    name: 'meson-g12a-sei510'
+    mach: amlogic
+    class: arm64-dtb
+    boot_method: uboot
+    flags: ['little_endian']
+    filters:
+      - blacklist: {defconfig: ['allnoconfig', 'allmodconfig']}
+
+  meson-g12a-x96-max:
+    name: 'meson-g12a-x96-max'
+    mach: amlogic
+    class: arm64-dtb
+    boot_method: uboot
+    flags: ['little_endian']
```

Comment:
`little_endian` is not used as a flag anywhere, it doesn't really hurt to add it but it's assumed to be little endian by default on arm* platforms unless `big_endian` is specified.  So it's probably better to not introduce `little_endian` now, and just remove the `flags:` attribute.

Reply:
Will apply these changes.